site/pages/react/react-hooks.mdx

```diff
@@ -1,153 +1,73 @@
 
 :::tip
 Tailwind CSS is a required dependency for using Alchemy Account Kit UI components. However, Alchemy Account Kit hooks function independently and do not require Tailwind.
 :::
 
-### Email authentication
+## Available Authentication Methods
 
-```tsx twoslash
-import React from "react";
-import {
-  type UseAuthenticateResult,
-  useAuthenticate,
-} from "@account-kit/react";
-
-// This examples uses email authentication
-// you can also use passkeys if the user has one created
-export default function MyLoginPage() {
-  const { authenticate, isPending } = useAuthenticate();
-  const [email, setEmail] = React.useState("");
-
-  return (
-    <div>
-      <input
-        type="email"
-        value={email}
-        onChange={(e) => setEmail(e.target.value)}
-      />
-      <button onClick={() => authenticate({ type: "email", email })}>
-        Login
-      </button>
-    </div>
-  );
-}
-```
+Account Kit supports several authentication methods that you can implement with custom UI. Each method has its own dedicated page with detailed implementation instructions, code examples, and specific parameters for the authentication hooks:
 
-### Social Login
+- [Email OTP](/react/login-methods/email-otp) - One-time password sent via email
+- [Email Magic Link](/react/login-methods/email-magic-link) - Authentication links sent via email
+- [Social Login](/react/login-methods/social-login) - Authentication with providers like Google and Facebook
+- [Custom Social Providers](/react/login-methods/social-providers) - Add custom OAuth providers via Auth0
+- [Passkey Signup](/react/login-methods/passkey-signup) - Create accounts with passkeys
+- [Passkey Login](/react/login-methods/passkey-login) - Authenticate with existing passkeys
 
-You can allow uers to sign in with an OAuth provider like Google, Facebook, or Auth0. There are two login flows that are possible: redirect and popup. The redirect flow redirects the user to the auth provider's login page, then back to a specified page in your application when done. The popup flow opens the auth provider's login page in a popup window without leaving your application, then updates your application when done. If using popup, you need to set the `enablePopupOauth` field to `true` in the [config](/react/quickstart).
+:::info
+**Visit each method's dedicated page** for specific implementation details, including the exact parameters to use with the `useAuthenticate` hook for that authentication method.
+:::
 
-For the authentication methods that Alchemy supports, authenticate by doing the following
+## Core Hooks for Custom UI
+
+The following section provides an overview of the main hooks you'll use when implementing custom authentication UI. These hooks are the foundation for all authentication methods, but their specific usage and parameters vary depending on the authentication method you choose.
```

Comment:
I just added some hooks below that a dev will likely end up using.  If there is anything you think I should add / remove from this, let me know